administrator/modules/mod_menu/src/Menu/CssMenu.php

```diff
@@ -236,7 +236,7 @@ protected function check($node, Registry $params)
 
             $table->load(['menutype' => $menutype]);
 
-            $menutype = $table->get('title', $menutype);
+            $menutype = isset($table->title) ? $table->title : $menutype;
```

Comment:
```suggestion
            $menutype =  $table->title ?? $menutype;
```
can't we do this? even if I don't it in trim functions I think here it's more readable then isset?

Reply:
I chose what is in the LegacyPropertyManagementTrait to prevent any unwanted change at all. I'm all for codereview, but I didn't want to block this PR by adding another change to it which might make it controversial in any way.